pkg/config/config.go

```diff
@@ -168,8 +169,51 @@ type LimitConfig struct {
 }
 
 // PodLogLimits limits the scope of response when getting logs from pods.
-// It exposes a subset of k8s.io/api/core/v1/PodLogOptions.
+// The scope of pods is defined by (Namespaces OR SonobuoyNamespace OR FieldSelectors) AND LabelSelector
+// Then for each pod, the format and size of logs is defined by other fields, e.g. SinceSeconds
 type PodLogLimits struct {
+	// A regular expression for the targeted namespaces, follows the format of Golang Regexp
+	Namespaces string `json:"Namespaces" mapstructure:"Namespaces"`
+
+	// If set to true, take into account the pod logs from Sonobuoy's namespace, i.e. Config.Namespace
+	SonobuoyNamespace *bool `json:"SonobuoyNamespace" mapstructure:"SonobuoyNamespace"`
```

Comment:
This is a `*bool`, is that just so that if nil we use the default value (which is true)? Looking at the definition like this does make it seem odd. (5% more of me is wondering if we should make that change to this field so the default is false and the field represents to _skip_ the sonobuoy namespace)

Reply:
The default value is handled at:

pkg/config/config.go: func New() *Config {
        ...
        cfg.Namespace = DefaultNamespace
        cfg.Limits.PodLogs.SonobuoyNamespace = new(bool)
	*cfg.Limits.PodLogs.SonobuoyNamespace = true
}

A nil value will be overwritten by this default value when doing config merge.
The same logic as cfg.Namespace, whose default value is ".*"